libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -443,24 +524,38 @@ export class ModelOverview extends React.Component<
         // remove unselected feature
         newlySelectedFeatures.splice(unselectedFeatureIndex, 1);
       }
+      const numberOfContinuousFeatureBins: {
```

Comment:
this looks like a lodash.mapValue should do

Reply:
I'm not sure I follow. Do you mean the data structure? lodash's mapValues is a function, isn't it?

Reply:
Will address this separately due to needing this PR merged urgently.